ncat/ncat_connect.c

```diff
@@ -294,7 +294,7 @@ static void connect_report(nsock_iod nsi)
             loguser("Connected to %s.\n", peer.un.sun_path);
         else
 #endif
-            loguser("Connected to %s:%hu.\n", inet_socktop(&peer),
+            loguser("Connected to %s:%d.\n", inet_socktop(&peer),
```

Comment:
The compiler did not announce me the other bad formats, but it seems to be just because of the #ifdef and #endif conditions, thought the compiler will not interpret them. I corrected the other bad formats specifiers. Thanks again for your time!